### ISSUE TITLE
[!] Error Occured

### ISSUE THREAD

**Issue by JohnCafe** [OsandaMalith/LFiFreak#1]
Hi, 

Everytime I try to execute a command, I get the message "[!] Error Occured"
(Auto test > PHP://input > Execute command > uname -a)

OS : KaliLinux 1.1.0

**Comment by OsandaMalith** [comment]
Hi, can u send me your file or the site you were testing? to reproduce the issue.

**Comment by JohnCafe** [comment]
So : 
localhost/dvwa
localIP/dvwa
http://www.niedax.de/page/index.php?page=niedax-group&lang=eng
And many others... I get this error everytime :S

Thnx !

**Comment by OsandaMalith** [comment]
Is that site vulnerable to LFI?

**Comment by JohnCafe** [comment]
dvwa is, you are using it in your demo.
PS : I really like your git !

**Comment by OsandaMalith** [comment]
Thanks :) btw did you pass the cookie values? Can you send me a full screenshot of the inputs you give?

**Comment by JohnCafe** [comment]
OFC, here it is : 
http://hpics.li/b76a82a

**Comment by OsandaMalith** [comment]
You are typing wrong. It should be uname -a

**Comment by JohnCafe** [comment]
Yes, I feel kind of stupid to miss it on the screenshot, it's the first time I type it this way... Anyway here is the screenshot with the correct command : 
http://hpics.li/0c7f97c
I can assure you that I know to type in a shell, even if I made a stupid mistake :-/

**Comment by OsandaMalith** [comment]
Do you use teamviewer? I will have to visit your PC and check for the issue. BTW check for other options. Does the reverse/bind shells work?

**Comment by yasoob** [comment]
Providing remote access to a hacker? No! Sorry :D :+1:
